src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
```

Comment:
The `href` attribute is set to `#`, which is a placeholder. Consider updating it to point to the actual URL or route for the logo link.